### DISCUSSION TITLE
Wave server doesn't start automatically with client

### DISCUSSION THREAD

**Discussion by AWRDev** [h2oai/wave#2206] [Q&A]
So, as I understand when I run my app with ```wave run app``` in a venv that has h2o-wave installed it should automatically start waved.exe then execute ```app.py```. But for me I have to run waved.exe manually before ```wave run app```

**Comment by mturoci** [comment]
Pip probably downloaded platform agnostic wheel. Can you share pip install output so that we can check?

**Reply by AWRDev** [reply]
Honestly, I don't quite know what is "pip install output". Like, I didn't install h2o-wave today to have a terminal open and just copy the output. Or am I getting this wrong?

**Reply by mturoci** [reply]
You can reinstall. So I would like to see the output (text in terminal) of running `pip install h2o-wave` within your virtual env.
